lib/Checkout/Transfers/TransfersClient.php

```diff
@@ -19,11 +19,11 @@ public function __construct(ApiClient $apiClient, CheckoutConfiguration $configu
 
     /**
      * @param CreateTransferRequest $transferRequest
-     * @param null $idempotencyKey
+     * @param string|null $idempotencyKey
      * @return array
      * @throws CheckoutApiException
      */
-    public function initiateTransferOfFunds(CreateTransferRequest $transferRequest, $idempotencyKey = null)
+    public function initiateTransferOfFunds(CreateTransferRequest $transferRequest, string $idempotencyKey = null)
```

Comment:
For backward compatibility please remove the `string` type in this line, I think in line 22 is enough for phpstan